pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -321,10 +327,12 @@ def _check_use_sequence_for_iteration(
 
     @utils.check_messages("consider-using-f-string")
     def visit_const(self, node: nodes.Const) -> None:
-        if node.pytype() == "builtins.str" and not isinstance(
-            node.parent, nodes.JoinedStr
-        ):
-            self._detect_replacable_format_call(node)
+        if self._py36_plus:
+            # f-strings require Python 3.6
```

Comment:
```suggestion
            # f-strings require py-version >= 3.6
```

Reply:
It also requires to have a Python interpreter greater than 3.6 (pylint won't run otherwise anyway.) but theorically we might need to specify both.